specs/windows/windows_eventlog.table

```diff
@@ -0,0 +1,23 @@
+table_name("windows_eventlog")
+description("Table for querying all recorded Windows event logs.")
+schema([
+    Column("channel", TEXT, "Source or channel of the event", required=True),
+    Column("datetime", TEXT, "System time at which the event occurred"),
+    Column("task", INTEGER, "Task value associated with the event", additional=True),
+    Column("level", INTEGER, "Severity level associated with the event", additional=True),
+    Column("provider_name", TEXT, "Provider name of the event"),
+    Column("provider_guid", TEXT, "Provider guid of the event"),
```

Comment:
Is `provider_name` the application name? That's a think I usually want to search by.

Reply:
Thanks for pointing this. I will add the constraint to filter events with `provider_name`. 

I also have changes adding a hidden `xpath` column that can be used to pass the complex query string extracted from the Event viewer to filter the events. Should I add the changes to this PR?

Reply:
I think it's reasonable to add it to this PR.

Reply:
I am a fan of having smaller commits so if the new `xpath` work fits nicely into a follow up PR, that is my preference.